README.md

```diff
@@ -37,6 +37,24 @@ repositories {
 }
 ```
 
+If you start with a new Android Studio project you have to add the `maven` block inside the `settings.gradle` file, e.g.:
+
+```groovy
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        jcenter() // Warning: this repository is going to shut down soon
+        maven {
+            url "https://maven.mozilla.org/maven2"
+        }
+    }
+}
+rootProject.name = "MyApp"
+include ':app'
```

Comment:
The Mozilla maven repository  is already mentioned in the README. The other repositories may be app project specific (and jcenter shouldn't be required at all for anything in this repo). Especially the last lines that include and setup the app module. :)

Reply:
Yes this was thought as an example. I have tried to use mozilla components in a completely new empty android project (within android studio). Adding the maven repo was not possible with the mentioned approach, it had to be added to the `settings.gradle`.